### PR TITLE
Update vpc.md

### PR DESCRIPTION
**Pull request by ramonguiu**
# Description

Add further clarification that connections are only possible from The Customer VPC to the Timescale VPC but not the other way around.

### CODE REVIEW

File: use-timescale/vpc.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ Your apps run inside your AWS Customer VPC, your Timescale Services always run
 inside the secure Timescale VPC. You control secure communication between apps in
 your VPC and your Timescale Services using a dedicated Peering VPC. The AWS PrivateLink connecting 
 Timescale VPC to the dedicated Peering VPC gives the same level of protection as using a direct AWS PrivateLink connection. 
-It only enables communication between your Customer VPC and your Timescale Services.
+It only enables communication to be initiated from your Customer VPC to Timescale Services running
```
Comment (billy-the-fish) [inline use-timescale/vpc.md:29]:
```suggestion
This enables you to initiate unidirectional communication from your Customer VPC to Timescale Services running
```

File: use-timescale/vpc.md [hunk 2]
```diff
@@ -26,7 +26,8 @@ Your apps run inside your AWS Customer VPC, your Timescale Services always run
...
+in the Timescale VPC but not the other way around.
```
Comment (billy-the-fish) [inline use-timescale/vpc.md]:
```suggestion
in the Timescale VPC. Timescale cannot initiate communicate with your VPC. 
```

**Review by billy-the-fish — CHANGES_REQUESTED** [review]
To make this double clear.

File: use-timescale/vpc.md (hunk 1 above)
Comment (ramonguiu) [reply use-timescale/vpc.md:29]: @billy-the-fish the suggestion leads to confusion. The communication will go in both directions (you make requests and get data back). But the communication can only be initiated (i.e. opening the connection) from the Customer VPC.

**Review by billy-the-fish — APPROVED** [review]

**Pull request merged**